charts/latest-v2/azuredisk-csi-driver/values.yaml

```diff
@@ -149,6 +149,13 @@ schedulerExtender:
       monitor:
         enabled: false
   servicePort: 8889
+  resources:
+    requests:
+      cpu: 150m
```

Comment:
hi @nearora-msft do you have some monitoring data for the requests and limits? and also is `300m` cpu limits enough?

Reply:
These are the max cpu and memory usage numbers from the latest Aurora test runs:

20K Stateless tests:
Max memory: 40MB
Max cpu: 139m

1K stateful tests:
Max memory: 152MB
Max CPU: 24m

303 Disks/ 303 Vm test:
Max Memory : 152
Max CPU: 24

I'll also add the link to the dashboards in the PR that I have on aks-rp